<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="JCPSI5U3D" ignored="0" name="BDD Test" tests="1" failures="1" timestamp="05 Dec 2019 09:03:10 GMT" time="0.287" errors="0">
  <testcase name="SampleTest" time="0.186" classname="com.qmetry.qaf.automation.step.client.Scenario">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at .When search for(scenarios/suite1.bdd:7)
	at qaf.example.steps.StepsLibrary.test(StepsLibrary.java:28)
	at com.qmetry.qaf.automation.step.JavaStep.doExecute(JavaStep.java:150)
	at com.qmetry.qaf.automation.step.BaseTestStep.execute(BaseTestStep.java:146)
	at com.qmetry.qaf.automation.step.StringTestStep.execute(StringTestStep.java:126)
	at com.qmetry.qaf.automation.step.client.Scenario.execute(Scenario.java:174)
	at com.qmetry.qaf.automation.step.client.Scenario.scenario(Scenario.java:237)
]]>
    </failure>
  </testcase> <!-- SampleTest -->
</testsuite> <!-- BDD Test -->
